decidim-initiatives/spec/system/filter_initiatives_spec.rb

```diff
@@ -80,6 +80,7 @@
 
         within ".order-by__dropdown" do
           expect(page).to have_selector("ul[data-dropdown-menu$=dropdown-menu]", text: "Random")
+          sleep(2)
```

Comment:
As this is only for v0.27 and its going to be deprecated in some months from now, I'll let this slide 😉

Reply:
Yeah, for some reason, the click is too fast, and i had to add that to allow js to load. I do not understand why is appearing now, and not in the last 1 year or so.

Reply:
I don't know either and as its only in v0.27 I don't strongly care either 🙃